ci/requirements/doc.yml

```diff
@@ -41,6 +41,4 @@ dependencies:
   - sphinxext-opengraph
   - sphinxext-rediraffe
   - zarr>=2.10
-  - pip:
-      # relative to this file. Needs to be editable to be accepted.
-      - -e ../..
+  - xarray
```

Comment:
this is doing something else entirely: it installs `xarray` from `conda-forge`, and thus a released version. We *don't* want that, because it would make previews of the current state on this repository impossible.

Reply:
If installing from conda fixes the error you see in #9141 @Anu-Ra-g then it indicates that the reason you're seeing errors must be caused by some local changes you've made to the xarray repo. I would suggest first ensuring that you can build the documentation with no changes (i.e. on the `main` branch) before seeing what breaks with your changes.

Reply:
in this case I believe they just didn't fetch the tags, which means `setuptools-scm` didn't report the right version, and `cfgrib` complains about that.

Reply:
> I would suggest first ensuring that you can build the documentation with no changes (i.e. on the main branch) before seeing what breaks with your changes.

I tried building the docs from the `main` branch but I'm getting the same error.